ui/app/adapters/aws-credential.js

```diff
@@ -4,15 +4,21 @@ export default ApplicationAdapter.extend({
   createRecord(store, type, snapshot) {
     let ttl = snapshot.attr('ttl');
     let roleArn = snapshot.attr('roleArn');
+    let roleType = snapshot.attr('credentialType');
+    let method = 'POST';
+    let options;
     let data = {};
-    if (ttl) {
-      data.ttl = ttl;
+    if (roleType === 'iam_user') {
+      method = 'GET';
+    } else {
+      if (ttl) {
+        data.ttl = ttl;
+      }
```

Comment:
Oops! Good catch - will update that. /me shakes fist at JavaScript

Reply:
I changed this check, but coming out of the TTL picker it won't be zero because it'll also have the unit attached. Still better to check undefined though 😅